python/ray/_private/runtime_env/nsight.py

```diff
@@ -0,0 +1,132 @@
+import os
+import logging
+import subprocess
+import copy
+from typing import Tuple, List, Dict, Optional
+
+from ray._private.runtime_env.context import RuntimeEnvContext
+from ray._private.runtime_env.plugin import RuntimeEnvPlugin
+from ray._private.utils import (
+    try_to_create_directory,
+)
+
+default_logger = logging.getLogger(__name__)
+
+NSIGHT_DEFAULT_CONFIG = {
+    "-t": "cuda,cudnn,cublas,nvtx",
+    "-o": "'worker_process_%p'",
+    "--cudabacktrace": "all",
+    "--stop-on-exit": "true",
+}
+
+
+def check_nsight_script(nsight_config: Dict[str, str]) -> Tuple[bool, str]:
+    """
+    Function to check if the provided nsight options are
+    valid nsys profile options and if nsys profile is installed
+
+    The function returns:
```

Comment:
Please follow the same convention as other docstring. I.e.,

```
Args:
    nsight_config: blah blah. No reason to specify a type since it is already annotated with input
Returns:
    tuple of ~~
```

Reply:
seems not changed?